external/include/protocols/liblwm2m.h

```diff
@@ -71,44 +54,49 @@
 #define _LWM2M_CLIENT_H_
 
 #ifdef __cplusplus
-#define EXTERN extern "C"
 extern "C" {
-#else
-#define EXTERN extern
 #endif
 
 #include <stdint.h>
 #include <stddef.h>
 #include <stdbool.h>
-#include <sys/time.h>
+#include <time.h>
```

Comment:
I don't know why you modify this file. Is it updating of Wakaama from open source?

Reply:
I think it's the replace issue. I didn't change it. you can check <time.h> in original path "external/wakaama/core/liblwm2m.h". actually now we are trying to merge those two files in conflict. we will consider to the issue when we are merging it.